pygmt/tests/test_x2sys_cross.py

```diff
@@ -65,7 +66,7 @@ def test_x2sys_cross_input_file_output_dataframe():
         output = x2sys_cross(tracks=["@tut_ship.xyz"], tag=tag, coe="i")
 
         assert isinstance(output, pd.DataFrame)
-        assert output.shape == (14294, 12)
+        assert output.shape == (14338, 12)
```

Comment:
Still a little unsure why GMT 6.5.0's `x2sys_cross` returns more rows than GMT 6.4.0. Might need to investigate.